includes/Core/Modules/Module_With_Tag_Trait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Interface Google\Site_Kit\Core\Modules\Module_With_Tag_Trait
```

Comment:
Also :)
```suggestion
 * Trait Google\Site_Kit\Core\Modules\Module_With_Tag_Trait
```

Reply:
@aaemnnosttv Suggested this be `Trait` but it's still marked as an `Interface`. Why is this one marked resolved? 😅